docs/startup/download/index.md

```diff
@@ -40,6 +40,10 @@ Starting from 2016, Samza will begin requiring JDK8 or higher. Please see [this
  * [samza-sources-0.8.0-incubating.tgz](https://archive.apache.org/dist/incubator/samza/0.8.0-incubating)
  * [samza-sources-0.7.0-incubating.tgz](https://archive.apache.org/dist/incubator/samza/0.7.0-incubating)
 
+
+ ### Samza Tools
```

Comment:
This needs to be filled only after we release because we don't know the link to the samza tools till we release it right?

Reply:
Got it. It would to add some description on what's the intended usage of this bin tgz as well. i.e. user already have YARN/Kafka/ZooKeeper setup, and this binary tgz is to install some command line tools that user can run tests and demos w/o downloading and compiling the source code.